app/views/Settings.js

```diff
@@ -22,14 +21,18 @@ import LocationServices from '../services/LocationService';
 import { FEATURE_FLAG_SCREEN_NAME } from '../views/FeatureFlagToggles';
 import { GoogleMapsImport } from './Settings/GoogleMapsImport';
 import { SettingsItem as Item } from './Settings/SettingsItem';
+import {
+  isGPS,
+  settingsLoggingActiveText,
```

Comment:
What do you think about keeping these scoped in a module?

something like
```
import { Copy, Assets } from '../TracingStrategyAssets'
...

return (
  <View>
      <Image src={Assets.Onboarding1Backround} />
       <Text>{Copy.Onboarding1Title}</Text>
...
```

This makes is a little easier to keep undefineds and nulls from floating around and is tends to result in less churn in the imports.

Reply:
I decided to go with a function, so that I can import `useTranslation` and load these values dynamically. Let me know what you think.